src/components/layout/Layout.tsx

```diff
@@ -28,7 +29,8 @@ import {
   HOME_URL,
   LAYER_2_GRANTS_URL,
   MERGE_DATA_CHALLENGE_URL,
-  SEMAPHORE_GRANT_URL
+  SEMAPHORE_GRANT_URL,
```

Comment:
same as above, are all of these currently active? (or planned to be re-used in the future?)

Reply:
Some of these are not active, they are closed rounds but we keep showing the landing pages for them. For example, https://esp.ethereum.foundation/layer-2-grants